src/mlpack/tests/ann_dist_test.cpp

```diff
@@ -121,7 +122,7 @@ TEST_CASE("JacobianBernoulliDistributionLogisticTest", "[ANNDistTest]")
     }
 
     module.LogProbBackward(target, jacobianB);
-    REQUIRE(arma::max(arma::max(arma::abs(jacobianA - jacobianB))) <= 3e-5);
+    REQUIRE(arma::approx_equal(jacobianA, jacobianB, "both", 5e-3, 5e-3));
```

Comment:
Any particular reason why you changed the tolerance?
Shouldn't it be this:-
```suggestion
    REQUIRE(arma::approx_equal(jacobianA, jacobianB, "both", 3e-5, 3e-5));
```